.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-- "8"
```

Comment:
Is the support for v8 gone with the proposed changes?

Reply:
Nope. We could test all 3 versions. Should i go for it?

Reply:
10 plus is enough IMO

Reply:
10 is absolutely sufficient, but could you please add an engine block to the `package.json` with the supported versions?

Reply:
https://github.com/terrestris/react-geo-baseclient/pull/450